src/format/mod.rs

```diff
@@ -399,7 +460,7 @@ fn format_inner<'a>(
     time: Option<&NaiveTime>,
     off: Option<&(String, FixedOffset)>,
     item: &Item<'a>,
-) -> fmt::Result {
+) -> FormatResult<()> {
```

Comment:
I changed it for consistency with the localized function but that breaks the API. I'm not sure what is best

Reply:
nah there is way too much duplicated code, I will correct this

Reply:
since `format_inner` is not pub its API can change, sorry if I said something incorrect about that.

A new function `pub format_item_localized` and possibly localized variants of the functions that call `format_item` will be necessary.